src/app/app-routing.module.ts

```diff
@@ -2,14 +2,16 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 import { AppRoutePath } from './enums/app-route-path.enum';
-import { CoursesPageComponent } from './pages/courses-page/courses-page.component';
-import { LoginPageComponent } from './pages/login-page/login-page.component';
+import { LoginPageComponent } from './pages/auth/login-page/login-page.component';
+import { CourseAddPageComponent } from './pages/courses/course-add-page/course-add-page.component';
+import { CoursesPageComponent } from './pages/courses/courses-page/courses-page.component';
```

Comment:
you can create a `index.ts` to export all components from there. So that imports will be shortened

Reply:
Will do, thanks